debian/conffiles

```diff
@@ -1,9 +1,8 @@
-/opt/intelmq/var/lib/bots/modify/example/modify.conf
```

Comment:
On Mon, Jul 25, 2016 at 05:44:42AM -0700, AaronK wrote:
> > @@ -1,9 +1,8 @@
> > -/opt/intelmq/var/lib/bots/modify/example/modify.conf
> 
> is that now in example/ or in the other directory?
> Isn't it called exampleS ?

No, due to setup.py:30 it is indeed
/opt/intelmq/var/lib/bots/modify/example.

> We are unsure what you intend here...

The `conffiles` file merely declares configuration files.  While the
files in example/ will still be there they don't need to be treated
as configuration files by the package manager;¹ hence, the removal
of that line.

A copy of the default.conf example is installed as
/opt/intelmq/var/lib/bots/modify/modify.conf by the package.  This
file is actually used as a configuration file, and thus declared as
one here.

  ¹) We can simplify things, though.  Since the modify bot configs
     also get copied to the /usr/share/doc hierarchy the original
     location has become redundant.  We should probably get rid of
     the whole /opt/intelmq/var/lib/bots/modify/example directory.